interactions/client/bot.py

```diff
@@ -75,21 +77,25 @@ class Client:
     def __init__(
         self,
         token: str,
+        cache_limits: Optional[Dict[type, int]] = None,
+        intents: Intents = Intents.DEFAULT,
+        shards: Optional[List[Tuple[int]]] = None,
+        default_scope: Union[int, Snowflake, List[int, Snowflake]] = None,
+        presence: Optional[ClientPresence] = None,
+        _logging: Union[bool, int] = None,
+        disable_sync: bool = False,
         **kwargs,
     ) -> None:
         self._loop: AbstractEventLoop = get_event_loop()
-        self._http: HTTPClient = token
-        self._intents: Intents = kwargs.get("intents", Intents.DEFAULT)
-        self._shards: List[Tuple[int]] = kwargs.get("shards", [])
+        self._http: HTTPClient = token  # noqa
```

Comment:
You could make it Union with string

Reply:
It was left as the httpclient from when you moved it later into the starting process, so I just left it like that. I'll change it as soon as I'm able to